fava/template_filters.py

```diff
@@ -145,3 +145,12 @@ def uptodate_eligible(account_name):
     """Determine whether uptodate-indicators should be shown for an account."""
     return g.ledger.account_metadata(account_name).get(
         'fava-uptodate-indication') == 'True'
+
+
+def rendered_entries(entries):
+    """Render the entries provided."""
+    from fava.core.file import get_entry_slice
+
+    return [get_entry_slice(entry)[0] for entry in entries
```

Comment:
Can't this be done in the template (maybe a method needs to be added on FileModule.

Reply:
628813f